rust/rope/src/delta.rs

```diff
@@ -22,10 +22,16 @@ use subset::{Subset, SubsetBuilder};
 use std::cmp::min;
 
 enum DeltaElement<N: NodeInfo> {
+    /// Represents the text in a range of the base document. Includes beginning, excludes end.
     Copy(usize, usize),  // note: for now, we lose open/closed info at interval endpoints
     Insert(Node<N>),
 }
 
+/// Represents an entire new document as a sequence of sections copied from
```

Comment:
I agree to some extent, but the reason I changed it is that while the struct conceptually represents a change, something that confused me at first was that the `els` field actually represents the entire new document, just by copying things from the old one.

I've changed the comment to reflect both of these things better.